server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+const app = express()
+const port = 3001
```

Comment:
It's a good practice to use environment variables for configuration settings like the port number. This allows for greater flexibility in different environments. [important]